downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -35,6 +35,8 @@ import java.io.File
 import java.io.IOException
 
 abstract class GitBase : VersionControlSystem() {
+    override val movingRevisionNames = listOf("HEAD", "master")
```

Comment:
Do you plan the replace this with a list of the actual branches later?

Reply:
No. My original idea was to really leave this hard-coded.